implementations/rust/ockam/ockam_app/Cargo.toml

```diff
@@ -58,6 +58,7 @@ tokio = { version = "1.31.0", features = ["full"] }
 tokio-retry = "0.3"
 tracing = "0.1"
 tracing-subscriber = { version = "0.3.17", features = ["env-filter"], optional = true }
+ockam_transport_tcp = { path = "../ockam_transport_tcp", version = "^0.84.0" }
```

Comment:
We have a CI job that checks that the dependencies are ordered alphabetically (we use `dprint`). It should be right below `ockam_multiaddr`.